indico/modules/admin/notices_test.py

```diff
@@ -0,0 +1,44 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2021 CERN
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see the
+# LICENSE file for more details.
+
+import pytest
+from packaging.version import Version
+
+from indico.modules.admin.notices import NoticeSeverity, SystemNotice, SystemNoticeCriteria
+from indico.testing.util import extract_logs
+
+
+@pytest.mark.parametrize(('criteria', 'expected'), (
```

Comment:
Thoughts on adding test cases which test invalid criteria, invalid versions and invalid operators?

Reply:
I actually did that a few minutes ago, just not pushed ;)